tests/functional/api/BackbeatServer.js

```diff
@@ -839,6 +839,40 @@ describe('Backbeat Server', () => {
                 done();
             });
         });
+
+        it('should return a response even if redis data does not exist',
+        done => {
+            redis.keys('*:test:bb:*').then(keys => {
```

Comment:
Thinking out loud: I'm considering this part of the test setup, so thinking it would help to isolate the test case by using a `decribe` and `before` hook here.